web_widget_html_markdown/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+* HTML Will be lost when passing from Html to markdown
```

Comment:
Done but slightly different, now that we changed stuff:
* Implement conversion from HTML to Markdown, either through a JS or a Python module, instead of not allowing  it, or allowing it only after HTML is emptied.